### ISSUE TITLE
FEATURE REQUEST - Shuffle

### ISSUE THREAD

**Issue by gchingon** [HermesApp/Hermes#201]
as the title states, I'd like to see Shuffle button available. As on the pandora.com you can toggle shuffle and which stations you wish to listen to. would love to see this implemented.

**Comment by liuml07** [comment]
I must say that this will be a very good feature.

Do you need help? I could fork this if you're busy.

**Comment by nriley** [comment]
Yes, if you would like to help with this, we are both really busy and would appreciate the help. Let us know if you have any questions about the code, and thanks!

**Comment by winny-** [comment]
Isn't this the Quickmix radio? I thought Hermes has the ability to edit the Quickmix station (what stations to play), but it doesn't appear as so. So that needs to be added.

**Comment by dromrell** [comment]
First - Great app - Thanks!  As a developer I have several refining comments on this feature request.

Pandora native (Android & Browser) has shuffle mode that enables checklist of the stations included in the shuffle. To mimic this Hermes could add "shuffle" toggle button above station list (near Name/Date sort).  When shuffle is toggled ON the station list can becomes multi-selectable. I like how Pandora will highlight the station that is contributing current playing song (and if you unselect the currently playing station from the shuffle skips to next song). I also like how it remember selected channels in shuffle across clients (open in new browser recalls the shuffle selection).

An alternate approach (that I would love) is ability to have multiple shuffle lists based on named moods that I create (Exercise, Romance, Instrumental, etc).  I imagine a drop down select instead of the shuffle button ON/OFF toggle.  Pandora's interface wouldn't save multiple named moods/shuffles across the devices, but Hermes could push my current mood selection into Pandora shuffle list so my next devices would pick up my mood.  It could also have a default shuffle mood called QuickMix that retains the list of stations in the Pandora shuffle list (from other devices).

**Comment by fritzboyle** [comment]
this would be great. I absolutely use quick mix nearly every time I listen to pandora. I love this app but this feature would put it over the top as my favorite way to enjoy pandora. not sure how I can help other than as QA/feedback (i am not a programmer) but sure willing to learn/help/try.

**Comment by nriley** [comment]
I've renamed QuickMix to "Shuffle (QuickMix)" as a start. I haven't investigated whether it's possible to select which stations contribute to the shuffle from the API.
